### ISSUE TITLE
Alias of $ can be imported infinitely

### ISSUE THREAD

**Issue by lencioni** [trotzig/import-js#214]
If you set `$` to be an alias for `jquery`, and have the following file:

```js
$
```

running import on the `$` will give you

```js
import $ from 'jquery';

$
```

which is perfect. But if you run it again you get

```js
import $ from 'jquery';

import $ from 'jquery';

$
```

which is not right. This can go on for forever.

If I have other imports after the jquery one, then I can import them again and the import appears above the jquery import and stays after it as well.

I haven't looked into it yet, but I bet that we have a regex bug in the part where we are finding the block of current imports that chokes on `$`.

**Comment by lencioni** [comment]
It might also be a bug in parsing `ImportStatements` since that is part of that logic.

**Comment by lencioni** [comment]
Oh I'm pretty sure this was fixed by bd696a0da66af65dd9f05365eaf8297db17ba61d. I thought I was using the latest version when I filed this, but apparently I was not.

**Comment by trotzig** [comment]
Yeah, that should have fixed it.
